### ISSUE TITLE
FR Implement Basic Training Functionality

### ISSUE THREAD

**Issue by wizardling1** [ntdenley/CS-320-Minecraft-Bot#9]
Implement at least one optimizer, at one loss function (MSELoss), and being able to print a summary of the network being trained.
